client/src/App.js

```diff
@@ -1,10 +1,44 @@
 import React from 'react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import allComponents from './Routes/allComponents';
+
+const {
+  LandingPage,
+  Login,
+  Signup,
+  Home,
+  Header,
+  Footer,
+  MyApplications,
+  MyOffers,
+  NewOffer,
+  OfferDetails,
+  SavedOffers,
+  Profile,
+  PageNotFound,
+} = allComponents;
 
 function App() {
   return (
-    <div>
-      <h1>React App</h1>
-    </div>
+    <Router>
+      <>
+        <Header />
+        <Switch>
```

Comment:
I suggest to collect all private routes into one component with specific route starts with i.e. '/app'
thanks